services/ui-src/src/components/report/ReportPageWrapper.tsx

```diff
@@ -93,7 +94,7 @@ export const ReportPageWrapper = () => {
           </Box>
           {!currentPage.hideNavButtons && parentPage && (
             <>
-              <Divider></Divider>
+              {parentPage.index == 0 && <Divider></Divider>}
```

Comment:
Visually, only the first page needs a `<Divider>` because the rest of the pages have what looks like a Divider already because they're tables. So leaving the `<Divider>` in as it was before makes it appear as double lines - not matching the Figma design. 

I'm unclear on how sustainable this approach is considering the nested pages may not always be tables if we add more pages to other reports in the future.

| Required Measure Results | Stratified Measure Results |
| ------- | ------ |
| <img width="400" alt="table 1" src="https://github.com/user-attachments/assets/8c6bee61-3b79-431a-afd8-28702662bd54"> | <img width="400" alt="table 1" src="https://github.com/user-attachments/assets/c9b370ac-1591-4604-97a7-d358b990b8d8"> |

Reply:
I'm guessing the Divider here should be mandatory and elments like tables should skip it if they're last. Regardless, can you add a TO DO comment here, and a ticket to solidify the Divider behavior for our form controls vs elements in a page? It feels like our discussion here will scope creep otherwise.

Reply:
Done ✅